### DISCUSSION TITLE
Version for Google Home Mini (1st Gen)

### DISCUSSION THREAD

**Discussion by fightforlife** [justLV/onju-voice#6] [Q&A]
Hi there, very interesting project! I would love to replace my home minis.
Will there be a version/schematic for the original Google Home Mini 1st Gen?
I could not find a hint in the readme, or maybe they are just compatible?

Besr regards

**Comment by justLV** [comment]
Definitely not compatible, someone did a teardown [here](https://justlv.medium.com/google-home-mini-teardown-comparison-to-echo-dot-and-giving-technology-a-voice-c59a23724a26) you can reference for the PCB size ;)
